_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: default
+title: Create reports with the Reporting CLI
+nav_order: 25
+---
+
+# Create reports with the Reporting CLI
+
+You can create Dashboard reports programmatically with the Reporting CLI without using the OpenSearch Dashboards and the Reporting plugin. This provides automation for reporting that can be connected to email workflows.
```

Comment:
I believe that CSV requires the Reporting plugin to be installed. @rupal-bq to confirm.

Reply:
yes, reporting plugin should be installed for csv reports